joomla/joomla-cms
# [3.5] Remove com_messages from core new installs

Following the roadmap for Joomla 3.5: 
`Remove additional components and all associated pieces. ([...] com_messages).[...]`
http://developer.joomla.org/cms/roadmap.html

This will remove com_messages from the core new installs.
### how to test (first test)
- download my branche: https://github.com/zero-24/joomla-cms/archive/patch-13.zip
- install as normal joomla 3.4
- make sure com_messages is gone and no errors are there e.g. SQL or others.
### how to test (seccond test)
- install normal 3.4
- update with a update package (needs to be created and uploaded to joomla.org)
- see that nothing is changed and com_messages still work as expeced.
### Open questions
- what we need to do outsited of this PR?
- how can we create a update package and upload it to joomla.org? (for the seccond test)
### More information
- New com_messages repo: https://github.com/joomla-extensions/messages (i will open a PR to add the needed files to this repo. (based on com_weblinks repo)
- First PR is done more PRs against the new repo will follow.
